atm.js

```diff
@@ -1,21 +1,73 @@
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
```

Comment:
It's giving me this error:` Uncaught TypeError: Cannot read property 'innerHTML' of null`

This is telling me your checkingBalance variable is undefined. You currently don't have a div with an id of checking_balance in your html, which I'm assuming is what you were planning to do here. Also, since you are using jQuery, for consistency I would not mix jQuery with Vanilla JS